### PR TITLE
Add pytest custom exit code

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
@conda-forge/staged-recipes please review

File: recipes/pytest-custom-exit-code/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set name = "pytest-custom-exit-code" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/92/9d/e1eb0af5e96a5c34f59b9aa69dfb680764420fe60f2ec28cfbc5339f99f8/pytest-custom_exit_code-{{ version }}.tar.gz"
```
Comment (isuruf) [inline recipes/pytest-custom-exit-code/meta.yaml]: Please use the url as in the example recipe

Comment (marcelotrevisani) [reply recipes/pytest-custom-exit-code/meta.yaml]: That one?
``https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz``
it was not working for this recipe

Comment (marcelotrevisani) [reply recipes/pytest-custom-exit-code/meta.yaml]: https://pypi.io/packages/source/p/pytest-custom-exit-code/pytest-custom-exit-code-0.3.0.tar.gz

Comment (marcelotrevisani) [reply recipes/pytest-custom-exit-code/meta.yaml]: https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/26740569#L271

Do you have any hints in this case?

Comment (marcelotrevisani) [reply recipes/pytest-custom-exit-code/meta.yaml]: I think I know why, I will try to fix it, thanks

Comment (isuruf) [reply recipes/pytest-custom-exit-code/meta.yaml]: https://pypi.io/packages/source/p/pytest-custom-exit-code/pytest-custom_exit_code-0.3.0.tar.gz

Comment (marcelotrevisani) [reply recipes/pytest-custom-exit-code/meta.yaml]: Thanks!

**Comment by marcelotrevisani** [comment]
@conda-forge/staged-recipes please review

**Pull request merged**